types/uni-app/index.d.ts

```diff
@@ -0,0 +1,12 @@
+// Type definitions for non-npm package uni-app 1.0
+// Project: https://github.com/dcloudio/uni-app
+// Definitions by: DCloud <https://github.com/dcloudio>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+// TypeScript Version: 2.3
+
+import "./lib/app";
```

Comment:
Why are these needed? Aren’t they all already global? Is it just to satisfy the linter?